CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG.md
 
+## 1.0.1
+
```

Comment:
Isn't crails still in beta? Releasing v1.0.1 would mean it's stable.

Reply:
That's true...
Should I update the main branch and rebase in that case?